radio/src/main.cpp

```diff
@@ -30,7 +30,14 @@ uint8_t currentBacklightBright = 0;
 uint8_t requiredBacklightBright = 0;
 uint8_t mainRequestFlags = 0;
 
+//OW
+#if defined(TELEMETRY_MAVLINK_USB_SERIAL)
+const char STR_USB_MAVLINK[] = "USB Mavlink (VCP)";
```

Comment:
Could you please explain why this is necessary?

Reply:
connecting to usb with mavlink support, i.e., you can connect e.g. your PC or smarphone to the usb and talk mavlink, very handy

Reply:
That is understood, but would the normal USB serial be ok? Or is it necessary to have a particular string here?

Reply:
you somehow need a way to select that it should be linked with the mavlink router, which it normally would not be ... so, some setting somewhere is needed to distinguish ... doing it that way is just super convenient because you plug in the usb and select as you do anyhow for joystick or vcp 
TANSTAAFL

Reply:
I get that point, no problem with that! It's just, why needing a different name here? Or is that creating a second VCP (on top of the debug one, in case compiled with debug options). You could probably just activate VCP, and that's it. Support for it is already there, as you can see.

Reply:
???
I'm not sure you got the point: If you have two options among which to select and both are named the very same how would you then ever know which one you select ???

* usb
* usb

so, now please select the one which links you to mavlink

> Support for it is already there, as you can see.
yes, and I'm obviously using it, don't I

I'm frankly totally lost

Reply:
@olliw42 It's not usb & usb, but Joystick and Mass Storage at the moment. What the MAVLink extensions need, is virtual serial port (VCP, or officially CDC). What @raphaelcoeffic says is that you already have a VCP there, e.g. if you build with DEBUG and CLI options set (vanilla OTX/ETX) and then hook the top USB-C to PC, you have three options pop-up. The serial option, that you need for MAVLink Routing via top USB-C is already there, so no need to redefine it with own name. I hope I was able to convey the meaning and clear up the issue a bit. Best!

Reply:
come on guys
we are talking about 
* VCP for serial
* VCP for serial with mavlink connection

so, make it 

* vcp
* vcp

if you want but the argument does not change at all

of course there can be situations, depending on flags, there only one of them might be available, it might even not be available at all LOL, but really, you want to change names depending on that, so that users need to know what their firmware ??? One could do as you suggest, all I'm saying is that it's IMHO short sighted.

Reply:
I don't want to change names at all! this it it ;-) These names are purely informational and not required for the function.